gef.py

```diff
@@ -10945,11 +10955,11 @@ def __init__(self, alias, command, completer_class=gdb.COMPLETE_NONE, command_cl
         gef.session.aliases.append(self)
         return
 
-    def invoke(self, args, from_tty):
+    def invoke(self, args, from_tty) -> None:
         gdb.execute("{} {}".format(self._command, args), from_tty=from_tty)
         return
 
-    def lookup_command(self, cmd):
+    def lookup_command(self, cmd: str) -> Optional[Tuple[str, Any, Any]]:
```

Comment:
the first Any would also be `ClassVar` ideally.

Reply:
`TypeError: Plain typing.ClassVar is not valid as type argument`